### ISSUE TITLE
Cordova 9 support

### ISSUE THREAD

**Issue by almothafar** [appfeel/cordova-play-services-version-adapter#2]
I'm getting issue:

```
Executing script found in plugin cordova-play-services-version-adapter for hook "before_prepare": plugins\cordova-play-services-version-adapter\scripts\adapt-play-services-
versions.js
Using "requireCordovaModule" to load non-cordova module "q" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
CordovaError: Using "requireCordovaModule" to load non-cordova module "q" is not supported. Instead, add this module to your dependencies and use regular "require" to load
it.

```

And I see the fix pushed here: https://github.com/appfeel/cordova-play-services-version-adapter/commit/a272a8cb62594e0b7492132eaaa2aecc4de0cab7 (PR #1)

Any chance we get the fix version published? currently, it is 1.0.2 which is 3 months old.

**Comment by almothafar** [comment]
I've added the plugin using:

```
ionic cordova plugin remove cordova-play-services-version-adapter --force
ionic cordova plugin add https://github.com/appfeel/cordova-play-services-version-adapter

```
Which uses the fix pushed, and seems it is working, so publish is enough.

**Comment by kyoz** [comment]
I'v follow @almothafar way and the error changing to:

```
'Cordova Play Services Version Adapter' has detected an error. The following dependencies of 'Google Play Services' are not compatible:
You have an incompatibility of versions with the following dependencies of 'Google Play Services':
    - 'com.google.firebase:firebase-messaging:17.0.+'
    - 'com.google.firebase:firebase-analytics:15.0.0'

It doesn't exists a version of 'Google Play Services' that includes all these dependencies.
All dependencies of 'Google Play Services' must have the same version
The application may not compile
```

:cry:

**Comment by appfeel** [comment]
fixed in v1.1.0

**Comment by appfeel** [comment]
> I'v follow @almothafar way and the error changing to:
> 
> ```
> 'Cordova Play Services Version Adapter' has detected an error. The following dependencies of 'Google Play Services' are not compatible:
> You have an incompatibility of versions with the following dependencies of 'Google Play Services':
>     - 'com.google.firebase:firebase-messaging:17.0.+'
>     - 'com.google.firebase:firebase-analytics:15.0.0'
> 
> It doesn't exists a version of 'Google Play Services' that includes all these dependencies.
> All dependencies of 'Google Play Services' must have the same version
> The application may not compile
> ```
> 
> 😢

This is an advertisement of the plugin telling that it can't find compatible versions. This is the main purpose of the plugin: if it can resolve it resolves, otherwise alerts user that it cannot resolve incompatibilities.
